src/services/ContractService/entities/Question.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
try to remove

Reply:
Still throwing errors, asking assignment or function call